src/main/scala/io/iohk/ethereum/healthcheck/HealthcheckService.scala

```diff
@@ -103,6 +108,48 @@ class HealthcheckService(
     mapPendingBlockHCResult
   )
 
+  private def mapVmCallHCResult(result: CallResponse): HealthcheckResult.MapperResult = {
+    None -> Map.empty
+  }
+
+  private def mapVmCallHCError(error: JsonRpcError): HealthcheckResult.MapperResult = {
+    Some(error.message) -> Map("code" -> error.code.toString, "type" -> error.productPrefix)
+  }
+
+  private def mapVmCallHCException(ex: Throwable): HealthcheckResult.MapperResult = {
+    Some(ex.getMessage) -> Map("type" -> ex.getClass.getSimpleName)
+  }
+
+  // STOP op code
+  private val evmByteCode = ByteString("0x00")
+
+  /*
+  contract test3 {
+    function test() public pure returns (uint) {
+        return 0;
+    }
+  }
+  */
+  //scalastyle:off
+  private val ieleByteCode =
```

Comment:
We could also include here a high level description of the code. So, in effect, what is this binary translated from?